app/frontend/src/store/modules/admin.js

```diff
@@ -0,0 +1,99 @@
+import { adminService } from '@/services';
+
+/**
+ * Admin Module
+ */
+export default {
```

Comment:
I'm half wondering if we need this new admin vuex module tree - feels like to me we can still do most operations directly in the components without requiring centralized state management for this. This is closely tied to the existing forms module already, so this potentially risks state duplication in the overall store.